### ISSUE TITLE
Who are maintainers for this repo

### ISSUE THREAD

**Issue by sparrell** [niemopen/oasis-open-project#2]
The governance pages in this repo describe the responsibilities of maintainers but do not include who the maintainers of this repo are. Shouldn't they be listed somewhere?

**Comment by OASIS-OP-Admin** [comment]
Agreed. We are managing maintainers via teams. We are recommending that maintainers be documented in the README with a link to the maintainer team for the repo. For this repo, it is https://github.com/orgs/niemopen/teams/niemopen-write

**Comment by OASIS-OP-Admin** [comment]
Chet will look into our practice on other projects and recommend a best practice. Name and organization.
